### ISSUE TITLE
Can not  Get  modules help information

### ISSUE THREAD

**Issue by shencan** [dominis/ansible-shell#39]
##  Ansible version
<pre>
[root@vm10-160-112-18 ansible-shell]# ansible --version
ansible 1.9.1
  configured module search path = None
</pre>

## error information
<pre>
root@* (10)[f:10]$ help copy
Traceback (most recent call last):
  File "/usr/bin/ansible-shell", line 378, in <module>
    AnsibleShell(options, args).cmdloop()
  File "/usr/bin/ansible-shell", line 136, in cmdloop
    cmd.Cmd.cmdloop(self)
  File "/usr/lib64/python2.7/cmd.py", line 142, in cmdloop
    stop = self.onecmd(line)
  File "/usr/lib64/python2.7/cmd.py", line 221, in onecmd
    return func(arg)
  File "/usr/lib64/python2.7/cmd.py", line 311, in do_help
    func()
  File "/usr/bin/ansible-shell", line 88, in <lambda>
    setattr(self, 'help_' + module, lambda module=module: self.helpdefault(module))
  File "/usr/bin/ansible-shell", line 332, in helpdefault
    oc, a = ansible.utils.module_docs.get_docstring(in_path)
ValueError: too many values to unpack
</pre>

**Comment by evgkrsk** [comment]
Seems like get_docstrings now returns tuple with 3 elements (doc, plainexamples, returndocs) and trivial one-line patch works around this issue:

```diff
--- /usr/bin/ansible-shell	2015-07-05 15:15:07.000000000 +0700
+++ /home/evg/bin/ansible-shell	2015-07-05 15:38:16.285786942 +0700
@@ -329,7 +329,7 @@
     def helpdefault(self, module_name):
         if module_name in self.modules:
             in_path = utils.plugins.module_finder.find_plugin(module_name)
-            oc, a = ansible.utils.module_docs.get_docstring(in_path)
+            oc, a, b = ansible.utils.module_docs.get_docstring(in_path)
             print stringc(oc['short_description'], 'bright gray')
             print 'Parameters:'
             for opt in oc['options'].keys():
```

**Comment by evgkrsk** [comment]
+
```diff
@@ -356,7 +356,7 @@ class AnsibleShell(cmd.Cmd):
 
     def module_args(self, module_name):
         in_path = utils.plugins.module_finder.find_plugin(module_name)
-        oc, a = ansible.utils.module_docs.get_docstring(in_path)
+        oc, a, b = ansible.utils.module_docs.get_docstring(in_path)
         return oc['options'].keys()
```

**Comment by shencan** [comment]
Thank  you

**Comment by dominis** [comment]
thanks guys it's now fixed in ec02e05
